### DISCUSSION TITLE
2016/03/30/beacon-light-purchased-can-anyone-help-identify-it

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#13] [Announcements]
# 2016/03/30/beacon-light-purchased-can-anyone-help-identify-it

http://localhost:4000/2016/03/30/beacon-light-purchased-can-anyone-help-identify-it

**Comment by jengalas** [comment]
Posted by Dave DuPree
February 3, 2017 at 10:16 pm
_Imported from WordPress_

This is an early-style Department of Commerce 24″ airway beacon. The specifications for this light were issued by the Aeronautics Branch, Airways Division / U. S. Lighthouse Service in 1928. This type of 24″ beacon was produced from 1928 to 1938-39. They were supplied to the federal airways under contract by Sperry Gyroscope, General Electric and Crouse-Hinds. This particular light appears to have been manufactured by Crouse-Hinds.

Without better photos, I can’t tell you much more. More often than not it’s impossible to determine the actual location where an old beacon served. It’s obviously a bit rough and missing some parts, but it is certainly worthy of restoration.
